src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParseTreeTest.java

```diff
@@ -311,4 +311,16 @@ public void testLeadingAsteriskAfterSeeTag() throws Exception {
         verifyJavadocTree(getDocPath("expectedLeadingAsteriskAfterSeeTagAst.txt"),
                 getDocPath("InputLeadingAsteriskAfterSeeTag.javadoc"));
     }
+
+    @Test
+    public void testUppercaseInPackageName() throws Exception {
+        verifyJavadocTree(getDocPath("expectedUppercaseInPackageNameAst.txt"),
+                getDocPath("InputUppercaseInPackageName.javadoc"));
+    }
+
+    @Test
+    public void testParagraph() throws Exception {
```

Comment:
Why paragraph is being tested in this PR?

Reply:
https://github.com/checkstyle/checkstyle/pull/4550#issuecomment-328360874

Reply:
That has to be at least a separate commit referring "increase coverage" issue

Reply:
@ps-sp, please also share cobertura report to support you comment

Reply:
[`master's coverage`](https://master-cobertura.netlify.com/)
[Coverage without `testParagraph` test in `JavadocParseTreeTest`](https://4408-decreased-coverage.netlify.com/)

Reply:
>That has to be at least a separate commit referring "increase coverage" issue

Done